performance/scaling_data_ingestion.rst

```diff
@@ -176,36 +176,38 @@ It may also be useful to create a sequence to generate a unique name for the sta
+Append distributed tables support COPY via the worker, by specifying the address of the master in a master_host option, and optionally a master_port option (defaults to 5432). COPY via the workers has the same general properties as COPY via the master, except the initial parsing is not bottlenecked on the master.
 
 ::
 
-    csql -h master-node -c "\\STAGE events FROM 'data.csv' WITH CSV"
+    psql -h worker-node-1 -c "\\COPY events FROM 'data.csv' WITH (FORMAT CSV, MASTER_HOST 'master-node')"
 
 
-An alternative to using \STAGE is to create a staging table and use standard SQL clients to append it to the distributed table, which is similar to staging data via the master. An example of staging a file via a worker using psql is as follows:
+An alternative to using COPY is to create a staging table and use standard SQL clients to append it to the distributed table, which is similar to staging data via the master. An example of staging a file via a worker using psql is as follows:
 
 ::
 
     stage_table=$(psql -tA -h worker-node-1 -c "SELECT 'stage_'||nextval('stage_id_sequence')")
     psql -h worker-node-1 -c "CREATE TABLE $stage_table (time timestamp, data jsonb)"
     psql -h worker-node-1 -c "\\COPY $stage_table FROM 'data.csv' WITH CSV"
-    psql -h master-node -c "SELECT master_append_table_to_shard(choose_shard('events'), '$stage_table', 'worker-node-1', 5432)"
+    psql -h master-node -c "SELECT master_append_table_to_shard(select_events_shard('events'), '$stage_table', 'worker-node-1', 5432)"
     psql -h worker-node-1 -c "DROP TABLE $stage_table"
     
-The example above again uses a choose_shard function to select the shard to which to append. To ensure parallel data ingestion, this function should balance across many different shards.
+The example above again uses a select_events_shard function to select the shard to which to append. To ensure parallel data ingestion, this function should balance across many different shards.
 
-An example choose_shard function belows randomly picks one of the 20 smallest shards or creates a new one if there are less than 20 under 1GB. This allows 20 concurrent appends, which allows data ingestion of up to 1 million rows/s (depending on indexes, size, capacity). 
+An example select_events_shard function belows randomly picks one of the 20 smallest shards or creates a new one if there are less than 20 under 1GB. This allows 20 concurrent appends, which allows data ingestion of up to 1 million rows/s (depending on indexes, size, capacity).
 
 ::
 
     /* Choose a shard to which to append */
-    CREATE OR REPLACE FUNCTION choose_shard(table_id regclass)
+    CREATE OR REPLACE FUNCTION select_events_shard()
```

Comment:
Any reason to change this function name? Also, maybe a name like 'choose_underutilized_shard()' might be more descrptive?